packages/gatsby/package.json

```diff
@@ -160,7 +160,8 @@
     "website"
   ],
   "license": "MIT",
-  "main": "cache-dir/gatsby-browser-entry.js",
+  "main": "cache-dir/commonjs/gatsby-browser-entry.js",
```

Comment:
I can also name this folder `cjs` but I honestly don't think it matters 🤷‍♂️ Both get the idea across.

Reply:
probably not super important, but i imagine the `browser` piece to this filename becomes less valuable since it would be unlikely to see that entry point target a browser

Reply:
Yeah, I don't think this is really worth changing! It'd require just a copy/move script e.g. `"postbuild:cjs": "mv cache-dir/commonjs/gatsby-browser-entry.js cache-dir/commonjs/gatsby-entry.js"` but not sure it's worth it.